src/v3/src/types/widget.ts

```diff
@@ -40,6 +39,7 @@ import { OktaSignInAPI } from './api';
 import { JsonObject } from './json';
 import { Modify } from './jsonforms';
 import { FormBag, RegistrationElementSchema } from './schema';
+import { DesignTokensType } from 'src/util/designTokens';
```

Comment:
I think this is VSCode's doing but we should use a relative import rather than the aliased one this generated